README.md

```diff
@@ -142,6 +142,35 @@ Some CSS rules in the Design System, such as the universal focus styling and the
 </div>
 ```
 
+4. Import the `DSProvider` for Chakra-based components
+
+As we start to integrate and compose DS components from Chakra components, we need to get the styles for those components. In order to do so, we need to wrap all the DS components that internally use with Chakra with a simple provider component.
```

Comment:
nit: looks like a typo in `wrap all the DS components that internally use with Chakra`